packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -603,6 +612,7 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
                             key={index}
                             onClick={this.handleComponentClick}
                             index={index}
+                            background={this.state.theme.background}
```

Comment:
This should error if theme is not provided, we should check to make sure theme is available, if not set this to null.

Reply:
Yep, this is expecting background as required, but it's optional. No value here should just pass null.